flask_itsyouonline.py

```diff
@@ -22,7 +22,7 @@
 -----END PUBLIC KEY-----"""
 
 
-def configure(app, organization, client_secret, callback_uri, callback_route, scope=None, get_jwt=False, offline_access=False, orgfromrequest=False, invalidate_session_timeout=300, verify=True):
+def configure(app, organization, client_secret, callback_uri, callback_route, scope=None, get_jwt=False, offline_access=False, orgfromrequest=False, invalidate_session_timeout=300, verify=True, org_is_required=True):
```

Comment:
org_is_required is not a clear parameter. I suggest to use "members_only" instead.
```suggestion
def configure(app, organization, client_secret, callback_uri, callback_route, scope=None, get_jwt=False, offline_access=False, orgfromrequest=False, invalidate_session_timeout=300, verify=True, members_only=True):
```